cg-spec/editors_draft.html

```diff
@@ -375,6 +375,21 @@ <h2>Overview</h2>
 -->
 </pre>
 
+    </section>
+    <section>
+      <h2>Common misconceptions</h2>
+
+      <p>Many people come to RDF-star with wrong ideas about what it is or how to use it. This section lists and corrects the most frequent ones.
+
+      <p>RDF-star is <em>not</em> syntactic sugar for <a href="https://www.w3.org/TR/rdf11-mt/#reification">standard reification</a>. While the latter is a vocabulary fitting in the standard RDF model (abstract syntax), RDF-star extends that model (see <a href="#concepts"></a>) with a new construct, namely <a>quoted triples</a>. Note also that the two are not mutually exclusive: standard reification can still be used in conjunction with RDF-star.
```

Comment:
> standard reification can still be used in conjunction with RDF-star

If anyone's got some compelling examples of this, it could be nice to add a section. (someday, not in this PR)

Reply:
I guess my point was to say: "if you are using standard reification, you don't need to change anything to migrate to RDF-star, your reifications will still "work" as before."

Are there concrete use-cases for using std reification together with quoted triples, beyond supporting legacy RDF? I am not entirely sure, because RDF-star offers a more compact way to express the same thing. With the following property:
```
:reificationOf a rdf:Property, rdf-star:TransparencyEnablingProperty ;
    rdfs:domain rdf:Statement ;
    rdfs:range rdf-star:Triple.
```
One could replace
```
[] a rdf:Statement ;
   rdf:subject _:s ;
   rdf:predicate :p ;
   rdf:object "o".
```
with
```
[] :reificationOf << _:s :p "o" >>.
```